### ISSUE TITLE
Loading cancer static tables

### ISSUE THREAD

**Issue by awm33** [cognoma/core-service#27]
Ideally some sort of script that can load the data on the local machine and in production.
- genes - comes django-genes
- organisms - comes django-organisms
- diseases
- samples
- mutations

**Comment by awm33** [comment]
https://bitbucket.org/greenelab/django-genes
https://bitbucket.org/greenelab/django-genes/src/a5bde519a8d3899e4edfe213cc8ff58a8d430e3f/genes/management/commands/genes_load_geneinfo.py?at=default&fileviewer=file-view-default

**Comment by cgreene** [comment]
I highly recommend a management command. If we don't mind the pandas dependency, you could load the CSV files with that and then add each row to the database via a management command.

**Comment by cgreene** [comment]
relevant docs: https://docs.djangoproject.com/en/1.10/howto/custom-management-commands/

**Comment by awm33** [comment]
I believe @stephenshank is giving this a try. Github won't let me assign someone without write access

**Comment by dhimmel** [comment]
Okay cool. @stephenshank you should open a pull request early on (while the feature is still incomplete), so we can provide incremental advice. Just make sure to work off of a branch that is not `master`. I have the sense that loading the cancer data could be easy if we do it right but burdensome if we do not.

**Comment by stephenshank** [comment]
Unfortunately at the moment I am struggling with Postgres. I've ran the commands on the `core-service` `README`. When I try to migrate, I get the following error:

`django.db.migrations.exceptions.MigrationSchemaMissing: Unable to create the django_migrations table (no schema has been selected to create in
LINE 1: CREATE TABLE "django_migrations" ("id" serial NOT NULL PRIMA...
                     ^
)
`

I am getting a similar error when trying to replicate the Django tutorial with Postgres (which I have done using sqlite and MySQL). I'm on a Mac and have installed postgres using Homebrew, which may not be ideal.

@awm33 do these errors indicate what could be going wrong? Sorry if I am posting in the wrong place, perhaps I should make a new issue.

**Comment by awm33** [comment]
@stephenshank Is the error coming from this repo or the django tutorial?

Does the cognoma schema exist? Did you change the settings? Sounds like this line is not in place https://github.com/cognoma/core-service/blob/master/cognoma_site/settings.py#L71

You could also try running these permissions commands https://github.com/cognoma/core-service/blob/master/api/test/runner.py#L8

**Comment by stephenshank** [comment]
@awm33 The above error is from `core-service`, running as described in the `README`.

I have not changed the settings. The line you referenced is in place.

I just tried dropping the schema and recreating with the command you referenced. I had to drop, since when I tried to create, postgres complained that it already existed. This makes me think the `README` commands worked. I still get the same error.

Thanks for your suggestions. I will try again on a Linux machine and report back (if it does not work, I will open up a new issue to reduce clutter on this one).
